### PR TITLE
Update to 1.6.0, fix template, license info

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,11 +37,17 @@ requirements:
...
     - pytz
     - six >=1.9
     - celery
     - geojson
     - pillow
+    - pydicom
+    - numpy
+    - pyjwt
+    - cryptography
+    - shutilwhich  # [not py3k]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Looks like this needs a feedstock. 😉

**Comment by pmlandwehr** [comment]
Waiting on [shutilwhich](https://github.com/conda-forge/staged-recipes/pull/1605)

**Comment by pmlandwehr** [comment]
`shutilwhich` is done rendering, so closing and reopening.

**Comment by jakirkham** [comment]
Instead of doing close/reopen on feedstocks, could you please restart the CIs directly? This should avoid starting unneeded CI builds with CIs that have already been determined to pass.

**Comment by pmlandwehr** [comment]
Yeah, I've switched over to doing that. It's been a bit of a timesaver.

**Pull request merged**